cms/templates/activation_active.html

```diff
@@ -1,5 +1,9 @@
 <%page expression_filter="h"/>
-<%! from django.utils.translation import ugettext as _ %>
+<%!
```

Comment:
**NOTE TO SELF:** Can these templates be removed? How would they ever appear?  Would they need tests?  Could remove new method `login_url_to_lms`, or should this be used in the header?

Reply:
I couldn't find these templates being used, so I removed them.